packages/orun/orun.0.0.1/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "Run benchmarks and measure performance"
+maintainer: "Shakthi Kannan <author@shakthimaan.com>"
+authors: "Stephen Dolan <stephen.dolan@cl.cam.ac.uk>"
+license: "MIT"
+homepage: "https://github.com/ocaml-bench/orun"
+bug-reports: "https://github.com/ocaml-bench/orun/issues"
+dev-repo: "git+https://github.com/shakthimaan/orun"
+depends:
+  [ "ocaml"
+    "dune" {>= "1.2"}
+    "re"
+    "cmdliner"
+    "yojson"
+]
+depexts: [
+  ["libdw-dev"] {os-family = "debian" | os-family = "ubuntu"}
+  ["elfutils-devel"] {os-family = "centos" | os-family = "fedora" | os-family = "oraclelinux"}
+  ["libdw-devel"] {os-family = "opensuse"}
+]
```

Comment:
I would highly recommend to move this into a new `conf-libdw` package. This would be much more maintainable.

Reply:
Any reason to have separate `conf-` packages? Wouldn't it be better to have a single source of truth and maintain all the dependencies in a single .opam file?

Reply:
> Wouldn't it be better to have a single source of truth and maintain all the dependencies in a single .opam file?

Precisely. Maintaining one conf-* is infinitely better and less maintenance than maintaining N orun packages.
I'll do it if you don't have time

Reply:
In opam-repository there is no such thing as "one single source of truth", everything is interconnected. The closest thing you can have to "one single source of truth" would be lock files and even that wouldn't be that accurate for depexts (e.g. see https://github.com/ocamllabs/opam-monorepo/issues/152)